### ISSUE TITLE
Is there an Example of Using Container?

### ISSUE THREAD

**Issue by EllisShen** [robinweser/react-layout-components#36]
Hi guys,

Does anyone know how to use the **Container** component? From the user guide, Container component has several Props like Margin, Border and Flow.., but it always shows error like `Unknown prop`Margin`on <div> tag. Remove this prop from the element.`... 

I tried two different syntax like:

```
                            <Container Margin="0 20 40 20 20">
                                <Box column>
                                    <text style={styles.title}>Sample1</text>
                                </Box>
                            </Container>
```

and

```
                            <Container marginTop={40} marginBottom={20} marginLeft={20} marginRight={20}>
                                <Box column>
                                    <text style={styles.title}>Sample2</text>
                                </Box>
                            </Container>
```

Both cases were failed and similar error messages shown up.

**Comment by ghost** [comment]
+1 Same over here. 

```
bundle.js:11747 Warning: Unknown prop `marginTop` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by Box)
    in Box (created by Container)
    in Container (created by GSSApp)
    in div (created by Page)
    in Page (created by GSSApp)
    in GSSApp (created by App)
    in MuiThemeProvider (created by App)
    in App
```

_Apart from showing the error it still seems to render properly._

Does a **Container** inherit from **Box**? Is it a **Box** with additional properties?

In general needs more examples!! Apart from that it looks great! :-)

**Comment by robinweser** [comment]
Heyhey @EllisShen, @kevart:

You're both right with those error messages. It is caused due to the changes to React coming with version 15.2. What we already do with <Box>, omitting props other than DOM attributes, we need to do this for Container and ScrollView as well. 
Luckily I already started adding these features and will push them today. The error messages are kind of warnings "only" as the Component still renders correctly. 

And yes, a Container is an extended <Box> to inherit all the flexbox stuff. As I am not using these components very often during the last month I was not able to further improve them as well as the documentation.

**Comment by EllisShen** [comment]
Hi @rofrischmann, 
Sorry I just saw your message. Did you already commit the change according to React update in ver 15.2 recently? I see the current version is still 3.0.1.

I very like the react-layout-component you developed, it saved me tons of time and very easy to use. Hopefully you got some time on improving the Container component.

**Comment by robinweser** [comment]
@EllisShen just released 3.0.2 including property omitting for Container, Page and ScrollView :P
